lib/models/hostapp.ts

```diff
@@ -227,6 +234,14 @@ const getHostappModel = function (deps: InjectedDependenciesParam) {
 							is_invalidated: false,
 						},
 					},
+					should_be_running__release: {
```

Comment:
How about creating a separate object, so that we don't retrieve all the data in both methods?
I might be missing something, but it looks like `should_be_running__release` isn't used anywhere in this file ?!?!

Reply:
yes sorry we can remove it, I was testing if it was possible to reuse the same method that handle the `owns__release` array  to also handle `should_be_running__release`and get the latest versions from there. But as far as I can see it's not exactly reliable